docs/modules/migrate/pages/dmt-start-migration-cluster.adoc

```diff
@@ -0,0 +1,27 @@
+= Start the Migration Cluster
+:description: The migration cluster is created by the custom Hazelcast distribution to migrate the data between your source and target clusters.
+
+{description}
+
+To start the migration cluster, complete the following steps:
+
+. Open a terminal
+. Navigate to the folder containing the extracted DMT package
+. Enter the following command:
++
+[source,shell]
+----
+HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start 
+----
+
+If the specified port is available, the cluster starts on that port. Otherwise, Hazelcast tries to find a free port as described in the xref:clusters:network-configuration.adoc#port[Port] section of the Networking topic. You can confirm the port used by the cluster in the logs displayed in your terminal.
+
+You can find the _migration.yaml_ file in the root folder of the DMT download package. If your logs show that the cluster starts on a different port to that specified in this file, you must update the `address` field to match the port number used.
+
+DMT uses this configuration file to connect to the migration cluster when running the migration.
+
+NOTE: The _migration.yaml_ file uses the same configuration options as the Hazelcast CLC. For further information on the options, refer to the https://docs.hazelcast.com/clc/latest/clc-config[Hazelcast CLC documentation^].
```

Comment:
This section may be updated in case named CLC configuration is to be used.

Reply:
Not changed for the moment, will revisit when approach is confirmed